ui/src/shared/constants/fluxFunctions.ts

```diff
@@ -626,16 +626,6 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
     link:
       'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/transformations/aggregates/increase',
   },
-  {
```

Comment:
Should this function be documented somewhere else now, to reflect its new home in the `influxdata/influxdb/v1` namespace?